packages/grafana-ui/src/options/builder/axis.tsx

```diff
@@ -81,6 +81,18 @@ export function addAxisConfig(
           { value: false, label: 'Off' },
         ],
       },
+    })
+    .addRadio({
+      path: 'axisColorFromSeries',
+      name: 'Color',
+      category,
+      defaultValue: false,
+      settings: {
+        options: [
+          { value: false, label: 'Auto' },
+          { value: true, label: 'From series' },
+        ],
+      },
```

Comment:
Since this is a boolean option - can it be turned into a checkbox, similarly to some options in Table panel:
<img width="291" alt="image" src="https://user-images.githubusercontent.com/2376619/176113397-4dd3e5e3-3d77-483f-8901-e2a01cd7031e.png">